source/agora/common/TransactionPool.d

```diff
@@ -149,6 +165,12 @@ public class TransactionPool
             db.execute("INSERT INTO tx_pool (key, val) VALUES (?, ?)",
                 hashFull(tx)[], buffer);
         }();
+
+        // insert each input information of the transaction
+        foreach (input; tx.inputs) {
```

Comment:
`{` on its own line. We follow this style guide: https://github.com/bpfkorea/agora/blob/v0.x.x/doc/Style.md